packages/app/src/app/components/Preview/DevTools/Terminal/index.tsx

```diff
@@ -130,7 +83,7 @@ class TerminalComponent extends React.Component<Props, State> {
     }));
   };
 
-  selectShell = (shellId: ?string) => {
+  selectShell = (shellId: string | undefined) => {
```

Comment:
@CompuIves You can just do the following instead

```suggestion
  selectShell = (shellId?: string) => {
```